.github/workflows/release-e2e.yaml

```diff
@@ -0,0 +1,58 @@
+---
+name: release-e2e
```

Comment:
nit: it looks like separate changes from a different PR were committed on the same branch as this one. I would recommend reverting the commit for this file on this branch and then squashing the three commits down to one (so just the bundle status message change remains)

Reply:
Not sure how my commit got messed up like that. Will definitely fix.